src/resources/app/pages/ViewCourse.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div class="screen mx-16">
+    <div
+      class="h-16"
+    ></div>
+    <div
+      class="flex flex-row flex-wrap items-start space-x-10"
+    >
+      <div 
+        class="m-0"
+      />
+      <div v-for="post in posts" v-bind:key="post.id">
+        <Card 
+        :course-name="post.name"
+        :room-num="post.room"
+        :teacher-name="post.createdBy"
+
+        id="course-card"
+        class="mb-10"
+      />
+      </div>
+      
+
+      <Add 
+        id="add-course-card"
+        class="mb-10"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+import Card from "../components/CourseCard.vue";
+import Add from "../components/AddCourseCard.vue";
+import api from "../../api/view";
+
+export default {
+
+  data() {
+    return {
+      posts: [
+        {
+            "id": 0,
+            "name": "Placeholder",
+            "subject": "Placeholder",
+            "room": "R-00",
+            "joinCode": "R00M00",
+            "createdBy": 1,
+            "createdAt": null,
+            "updatedAt": null
+        },
+      ],
+    }
+    // course: 'Software Engineering',
+    // room: 'A-1002',
+    // teacher: 'Pa Nofaldi'
+  },
+
+  methods: {
+    async getData() {
+      try {
```

Comment:
It would be nice to create a loading animation while awaiting for api request for better UX. You can utilize icon provided by vuetify and use tailwind for the animation. Example can be seen here:
https://github.com/uasoft-indonesia/badaso-lms-theme/blob/PBI-1-register/src/resources/app/pages/login.vue

Reply:
I'll do this later